controllers/release/adapter_test.go

```diff
@@ -1465,37 +1760,206 @@ var _ = Describe("Release adapter", Ordered, func() {
 		BeforeEach(func() {
```

Comment:
Couldn't this be AfterAll and BeforeAll instead of creating and deleting the pipelinerun between each run? Since no change is made before the creation

Reply:
Maybe but as we use AfterEach and BeforeEach in every single test to not condition tests within a when block, I kept it like that.